gtsam/hybrid/HybridGaussianFactor.h

```diff
@@ -37,6 +37,12 @@ class GTSAM_EXPORT HybridGaussianFactor : public HybridFactor {
   using This = HybridGaussianFactor;
   using shared_ptr = boost::shared_ptr<This>;
 
+  HybridGaussianFactor() = default;
+
+  HybridGaussianFactor(const KeyVector &continuousKeys,
+                       const DiscreteKeys &discreteKeys)
+      : Base(continuousKeys, discreteKeys) {}
```

Comment:
I'm a bit confused why we could construct a Gaussian factor here without any parameters eg. A, b?
Also, it seems inner_ is empty by using this constructor.

Reply:
Ah good catch. Let me verify this really quick.